Makefile

```diff
@@ -1,10 +1,13 @@
-.PHONY: depend up down
+.PHONY: depend set-region up down
 
 depend:
-	ansible-galaxy install -r ./provision/requirements.yml
+    ansible-galaxy install -r ./provision/requirements.yml
```

Comment:
What happened here? Have you checked this change didn't break `make`?

Reply:
I have checked it. It's ok :+1: